application-confluence-migrator-pro-ui/pom.xml

```diff
@@ -30,6 +30,13 @@
   <artifactId>application-confluence-migrator-pro-ui</artifactId>
   <name>Confluence Migrator Pro - UI</name>
   <packaging>xar</packaging>
+  <properties>
+    <!-- The list of documents that are not covered by the extension license. The user that installs the extension has
+      full access to these documents (view, edit, delete) without having a license, so use this property wisely. -->
+    <xwiki.extension.licensing.excludedDocuments>
+      ConfluenceMigratorPro.Code.PackagesStore
```

Comment:
Can't we create this page when the first package is uploaded? Normally the upload action creates the document if it doesn't exist already.